packages/eas-cli/src/project/android/gradleUtils.ts

```diff
@@ -30,7 +30,15 @@ export const DEFAULT_MODULE_NAME = 'app';
 export async function getAppBuildGradleAsync(projectDir: string): Promise<AppBuildGradle> {
   const buildGradlePath = AndroidConfig.Paths.getAppBuildGradleFilePath(projectDir);
   const rawBuildGradle = await fs.readFile(buildGradlePath, 'utf8');
-  return await g2js.parseText(rawBuildGradle);
+
+  // filter out any comments
+  // when comments are present, gradle-to-js fails to parse the file
+  const rawBuildGradleWithoutComments = rawBuildGradle
+    .split('\n')
+    .filter(line => !line.startsWith('//'))
```

Comment:
- do we need to `line.trim()`?
- does it also fail on multi-line comments (`/* comment */`?)

Reply:
>do we need to line.trim()?

yes 👍

Reply:
>does it also fail on multi-line comments (/* comment */?)

no

Reply:
>shall we add some simple test for this?

we can!